cpanfile

```diff
@@ -19,18 +19,27 @@ test_requires 'PadWalker';
 test_requires 'Test::Builder::Module';
 test_requires 'IO::String';
 test_requires 'Test::Perl::Critic';
+test_requires 'Test::Pod::Coverage';
 test_requires 'Perl::Critic::Utils';
 
-=cut
 feature 'assembly_mapping', 'Assembly mapper support' => sub {
   requires 'Algorithm::Diff';
   requires 'Tie::IxHash';
 };
 
 feature 'xref_mapping', 'Xref mapping pipeline' => sub {
-  requires 'Config::IniFiles';
```

Comment:
Config::IniFiles still used in Xref/DB.pm and xref_config2sql.pl

Reply:
This is redundant, though - see line 6 of this file.